Documentation/README_MRTKStandardShader.md

```diff
@@ -23,6 +23,21 @@ A custom material inspector exists for the MRTK/Standard shader called **MixedRe
 
 ![materialinspector](../Documentation/Images/MRTKStandardShader/MRTK_MaterialInspector.jpg)
 
+## Performance
+
+One of the primary advantages to using the MRTK Standard shader over the Unity standard shader is performance. The MRTK Standard Shader is extensible to only utilize the features enabled. However, the MRTK Standard shader has also been written to deliver comparable aesthetic results as the Unity Standard shader but at a much lower cost. One simple way to compare shader performance is via the number of operations that needs to be performed on the GPU. Of course, the magnitude of calculations may fluctuate by features enabled and other rendering configurations. But, in general, the MRTK Standard shader performs significantly less computation than the Unity Standard shader.
+
+Unity Standard shader statistics example
+
+![Unity Standard Shader Statistics](../Documentation/Images/Performance/UnityStandardShader-Stats.png)
```

Comment:
Did you need to commit this image too?

Reply:
Yes, it is already there under the performance page